app/src/main/java/com/kamwithk/ankiconnectandroid/ankidroid_api/MediaAPI.java

```diff
@@ -35,6 +35,7 @@ public MediaAPI(Context context) {
      */
     @SuppressLint("SetWorldReadable")
     public String storeMediaFile(String filename, byte[] data) throws IOException {
+        // TODO: investigate why filename gets a number attached to it, i.e. file.png -> file_123456789.png
```

Comment:
Is this number maybe just a unique identifier?

Reply:
Yes I think it's just a random unique ID, issue is that it's not accurate to how desktop Ankiconnect handles saving files. Desktop version saves files as the passed in filename.